app/src/main/AndroidManifest.xml

```diff
@@ -31,6 +31,13 @@ android:supportsRtl="true"
             android:exported="false"
             android:label="@string/title_activity_settings"
             android:theme="@style/AppThemeNew"/>
+        
+        <activity
```

Comment:
Dont add another entry point for the app, consider using android Fragments instead.